CI/macos/01_install_dependencies.sh

```diff
@@ -90,7 +90,7 @@ install_cef() {
     status "Set up dependency CEF v${1}"
     ensure_dir "${DEPS_BUILD_DIR}"
     unset _SKIP
-    
+
```

Comment:
Is that a stray tab or a removed tab? GitHub's diff view doesn't make it clear what it did here..

Reply:
Probably removed by .editorconfig.

Reply:
Discussed off-thread, and we'll leave this change in.